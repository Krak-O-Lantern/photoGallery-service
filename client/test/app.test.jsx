/* eslint-disable no-undef */
/* eslint-disable import/extensions */
import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from '../src/app.jsx';

describe('App component', () => {
  const wrapper = shallow(<App />);
  it('should render App component', () => {
    expect(wrapper.exists()).toBe(true);
  });

  describe('A suite example using Snapshot', () => {
    it('renders App component correctly', () => {
      const tree = renderer
        .create(<App />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
