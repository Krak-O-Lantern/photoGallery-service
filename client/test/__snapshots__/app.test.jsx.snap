// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component A suite example using Snapshot renders App component correctly 1`] = `
<div>
  <div
    className="sc-jSFkmK eqpWwz"
  >
    <h1
      className="sc-gKAblj jxIfqZ"
    >
      Luxury lodge with incredible views of Lake Washington
    </h1>
    <span>
      <img
        alt="rating star image"
        className="sc-iCoHVE jqvFOh"
        src="https://ghrsea12-fec.s3-us-west-2.amazonaws.com/sample/star.png"
      />
    </span>
    <span
      className="sc-fujyUd hwPare"
    >
      4.87
    </span>
    <br />
    <img
      alt=""
      className="sc-bdnylx ciqYQd"
      onClick={[Function]}
    />
    <img
      alt=""
      className="sc-gtssRu LSNQf"
      onClick={[Function]}
    />
    <img
      alt=""
      className="sc-dlnjPT iTvpQA"
      onClick={[Function]}
    />
    <img
      alt=""
      className="sc-hKFyIo iiBuWN"
      onClick={[Function]}
    />
    <img
      alt=""
      className="sc-eCApGN kpuqqu"
      onClick={[Function]}
    />
  </div>
</div>
`;
